server/src/controllers/eventController.service.ts

```diff
@@ -63,6 +62,45 @@ export const createStripeCoupon = async (discount: any, orderTotal: number) => {
   return stripeCoupon.id;
 };
 
+export const createStripePaymentIntent = async (
+  orderTotal: number
+) => {
+  const intentObject: JsonObject = {
+    currency: 'usd', // hardcode
+    payment_method_types: ['card_present'],
+    capture_method: 'automatic',
+    confirmation_method: 'automatic',
+    amount: orderTotal,
+    // off_session: false, // customer present
```

Comment:
The comments could be removed along with the commented out field if it's not being used

Reply:
Removed unecessary comments